devices/tuya.js

```diff
@@ -2496,6 +2496,58 @@ module.exports = [
         },
         whiteLabel: [{vendor: 'Hiking', model: 'DDS238-2'}, {vendor: 'TuYa', model: 'RC-MCB'}],
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_lsanae15']),
+        model: 'TS0601_din_2',
+        vendor: 'TuYa',
+        description: 'Zigbee DIN energy meter',
+        fromZigbee: [tuya.fzDataPoints],
+        toZigbee: [tuya.tzDataPoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [
+            tuya.exposes.switch(),
+            e.energy(),
+            e.power(),
+            e.voltage(),
+            e.current(),
+            exposes.enum('fault', ea.STATE, ['clear', 'over current threshold', 'over power threshold',
+                'over voltage threshold', 'wrong frequency threshold'])
+                .withDescription('Text of fault'),
+            exposes.binary('threshold1_state', ea.STATE, 'ON', 'OFF')
+                .withDescription('null - not set, OFF - just alarm, ON - relay will be off when threshold reached'),
+            exposes.text('threshold1_description', ea.STATE),
+            exposes.numeric('threshold1_value', ea.STATE)
+                .withDescription('Setup value on the device'),
+            exposes.binary('threshold2_state', ea.STATE, 'ON', 'OFF')
+                .withDescription('null - not set, OFF - just alarm, ON - relay will be off when threshold reached'),
+            exposes.text('threshold2_description', ea.STATE),
+            exposes.numeric('threshold2_value', ea.STATE)
+                .withDescription('Setup value on the device'),
+            exposes.binary('clear_fault', ea.STATE_SET, 'ON', 'OFF')
+                .withDescription('Clear the fault'),
+            exposes.text('meterid', ea.STATE).withDescription('Meter ID'),
+        ],
+        meta: {
+            tuyaDatapoints: [
+                [1, 'energy', tuya.valueConverter.divideBy100],
+                [3, null, null], // Monthly, but sends data only after request
+                [4, null, null], // Dayly, but sends data only after request
+                [6, null, tuya.valueConverter.phaseB], // voltage and current
+                [10, 'fault', tuya.valueConverterBasic.lookup({'clear': 0, 'over current threshold': 1,
+                    'over power threshold': 2, 'over voltage threshold': 4, 'wrong frequency threshold': 8})],
```

Comment:
Please make these snake case, e.g. `over power threshold` -> `over_power_threshold`

Reply:
did it